releasetasks/templates/shared/dummy_task.yml.tmpl

```diff
@@ -0,0 +1,34 @@
+{% macro common_extras(taskname, locales, platform) %}
+{% include "common_extras.yml.tmpl" %}
+{% endmacro %}
+
+{% macro task_notifications(taskname, failed=None, exception=None, artifact=None, completed=None) %}
+{% include "notifications.yml.tmpl" %}
+{% endmacro %}
+
+
+provisionerId: aws-provisioner-v1
+workerType: gecko-decision   # Fastest worker we have https://bugzilla.mozilla.org/show_bug.cgi?id=1318253#c3
```

Comment:
small nit: I guess {# #} would be the correct way to do it via official docs[1] but I see lots of references like this in the existing templates so I suppose we're alright.

[1]: http://jinja.pocoo.org/docs/2.9/templates/#comments